CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 1.7.0-beta.1
+
+## ✨ Features
+* Add `promoteId` option to use a feature property as ID for feature state ([#8987](https://github.com/mapbox/mapbox-gl-js/pull/8987))
+* Use `browser.devicePixelRatio` instead of directly calling `window.devicePixelRatio` to be able to override it ([#9063](https://github.com/mapbox/mapbox-gl-js/pull/9063)) (h/t [pakastin](https://github.com/pakastin)) 
+* Add a new constructor option to `mapboxgl.Popup`, `closeOnMove`, that closes the popup when the map's position changes ([#9163](https://github.com/mapbox/mapbox-gl-js/pull/9163))
+* Edit convert functions to output linear interpolation ([#9107](https://github.com/mapbox/mapbox-gl-js/pull/9107)) (h/t [Jo-IE](https://github.com/Jo-IE))
```

Comment:
This probably needs a bit more context, or it can be deleted, from reading the original issue I don't think it changes anything.

Reply:
Yeah, sorry I had a lot of issues to scan through and definitely didn't get all the details right. This was one of the ones I felt the most hazy about. No apologies needed thank you so much for this help! Can delete, have been leaning on the side of leaving in things with hattips.

Reply:
Also removing from style-spec changelog.